pandas/core/frame.py

```diff
@@ -7802,7 +7828,7 @@ def apply(
         )
         return op.get_result()
 
-    def applymap(self, func, na_action: Optional[str] = None) -> DataFrame:
+    def applymap(self, func: Callable, na_action: Optional[str] = None) -> DataFrame:
```

Comment:
Can this be typed more strictly than Callable? Subscripting Callable is much more useful to a reader if possible

Reply:
Yes - @jreback asked for this too. I'll put up a follow-on for this (to avoid expanding the scope of this PR)